lib/charms/layer/apache_bigtop_base.py

```diff
@@ -23,8 +24,8 @@ def __init__(self):
         self.bigtop_dir = '/home/ubuntu/bigtop.release'
         self.options = layer.options('apache-bigtop-base')
         self.bigtop_version = self.options.get('bigtop_version')
-        self.bigtop_base = Path(self.bigtop_dir) / self.bigtop_version
-        self.site_yaml = self.bigtop_base / self.options.get('bigtop_hiera_siteyaml')
+        self.bigtop_base = Path(self.bigtop_dir) / 'bigtop-%s' % self.bigtop_version
+        self.site_yaml = Path(self.bigtop_base) / self.options.get('bigtop_hiera_siteyaml')
```

Comment:
I actually like the addition of the redundant Path invocation. Makes it clearer exactly what is happening.